### DISCUSSION TITLE
摩托羅拉「無線充電」技術亮相！三公尺內可幫四台手機充電

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#822] [General]
https://3c.ltn.com.tw/news/45851
若充電範圍三公尺的無線充電器能普及的話
書桌上就不會有亂七八糟的充電線們了
充電地點也不再受限
而且一次能充電四台裝置等於大部分個家庭買一台就夠用了！
聽起來超棒的啊！

**Comment by Kotori0501** [comment]
感謝分享
一次能充四台裝置真的很便利，也不太需要用延長線同時衝手機或者平板，這樣我宿舍的書桌看起來就不那麼亂了，而且也支援三公尺的範圍，就不用把手機放在無限充電盤上了，最後希望能支援就手機就好了。

**Comment by bonny123865** [comment]
我覺得這種產品不錯欸，而且就不會因為充電線用太久接觸不良的問題了
而且還會自動避免讓電波影響身體，感覺也間接降低了風險，售價不貴的話感覺可以入手一個!!

**Comment by moonnight3503** [comment]
最近常看到關於無線充電爆炸或是損壞失靈的新聞，感覺無線充電還是一個尚未完全發達的技術，雖然可以一次充四台手機很方便，但希望在安全方面能有更多的保障

**Comment by ru0372572** [comment]
看起來好讚，但感覺還要很久才會出現在市面上

**Comment by Pietra1226** [comment]
太強了吧，這樣就不用因為要充電而被綁在書桌前或是插座前，可以到處活動，又可以讓手機充電。實在太方便了

**Comment by CPCpyt** [comment]
完全想不到，我對無線充電的概念還在觸碰式充電。
遠距充電這樣的概念真是讓人眼睛一亮，大為震撼。

**Comment by ruthchen1020** [comment]
這個技術是甚麼，電磁爐的鍋子會不會也發熱。

**Comment by StevenWanglolz** [comment]
聽起來來超級方便，不過感覺會有一些安全上的疑慮?希望在上市前能做足夠的安全測試!
